router.go

```diff
@@ -128,6 +128,32 @@ func createVersion(c echo.Context) (err error) {
 	return c.JSON(http.StatusCreated, ver)
 }
 
+func getPendingVersions(c echo.Context) (err error) {
+	if err = checkAuthorized(c); err != nil {
+		return err
+	}
+
+	editor := c.Param("editor")
```

Comment:
Its `QueryParam` (`Param` gets the elements defined in the routing layout, /plop/:editor/data for instance)